presentation/build.gradle

```diff
@@ -41,17 +41,15 @@ androidExtensions {
 }
 
 dependencies {
-    implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation 'androidx.appcompat:appcompat:1.1.0'
-    implementation 'androidx.core:core-ktx:1.1.0'
-    implementation "android.arch.lifecycle:extensions:$lifecycle_version"
-    implementation "android.arch.lifecycle:viewmodel:$lifecycle_version"
-    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:$coroutine"
+    api fileTree(dir: 'libs', include: ['*.jar'])
+    api "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    api 'androidx.appcompat:appcompat:1.1.0'
+    api 'androidx.core:core-ktx:1.1.0'
+    api "android.arch.lifecycle:extensions:$lifecycle_version"
+    api "android.arch.lifecycle:viewmodel:$lifecycle_version"
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:$coroutine"
-    implementation "javax.inject:javax.inject:$javaxInjectVersion"
-    implementation "com.google.dagger:dagger:$daggerVersion"
-    implementation "com.google.dagger:dagger-android-support:$daggerVersion"
+    api "com.google.dagger:dagger:$daggerVersion"
+    api "com.google.dagger:dagger-android-support:$daggerVersion"
```

Comment:
I think I know why you're using `api`, but this increases build time and it's not really advisable. I would say it's better to use `implementation` in each module, and if you want, you can extract all dependencies to a different gradle file and reuse the variables you'd assign to these dependencies.